frontend/src/app/libs/event-description/components/event-description-panel/event-description-panel.component.scss

```diff
@@ -1,29 +1,137 @@
+@import "variables";
+
+$appear-bottom-panel: 200px;
+
 .app-event-description-panel {
-	display: flex;
-	flex-direction: column;
-	flex-wrap: nowrap;
-	justify-content: space-around;
-	align-items: center;
-	width: 100%;
-	letter-spacing: 1.25px;
-	color: #000000;
+	background-color: #f8f8f8;
+
+	&__breadcrumb {
+		position: sticky;
+		top: $layout-column-width;
+		z-index: 2;
+	}
+
+	&__container {
+		display: flex;
+		align-items: flex-start;
+		justify-content: space-between;
+		padding: 0 $main-grid-unit * 22 0 $main-grid-unit * 22;
+	}
+
+	&__box-left {
+		width: 600px;
+		margin: $main-grid-unit * 5 0;
+	}
+
+	&__box-right {
+		width: 290px;
+		margin: $main-grid-unit * 5 0;
+	}
 
 	&__title {
-		font-weight: 500;
-		font-size: 60px;
-		line-height: 120%;
-		text-transform: capitalize;
+		@include text-h1();
+		color: $color-white-100;
+		margin-bottom:  $main-grid-unit * 2;
+	}
+
+	&__description {
+		@include text-p();
+		color: $color-white-100;
+		margin-bottom:  $layout-gutter;
+	}
+
+	&__take-part-button {
+		@include text-h4();
+		::ng-deep {
+			button {
+				padding: 12px $main-grid-unit * 2 !important;
+			}
+		}
+	}
+
+	&__take-part-button {
+		::ng-deep {
```

Comment:
Is there any way to avoid using ng-deep? :)

Reply:
Can I specify the component "encapsulation: ViewEncapsulation.ShadowDom"?